internal/version.go

```diff
@@ -30,7 +30,7 @@ package internal
 const (
 	// SDKVersion is a semver (https://semver.org/) that represents the version of this Temporal GoSDK.
 	// Server validates if SDKVersion fits its supported range and rejects request if it doesn't.
-	SDKVersion = "1.13.0"
+	SDKVersion = "1.13.1"
```

Comment:
This is just a temporary change for those trying this out to confirm they are on the right branch. This will be removed before merge.